### PR TITLE
2.5.6

### PR DESCRIPTION
**Pull request by sea-bass**
### Description

This PR commits the version bumps and generated (and cleaned up) changelogs in preparation for a Humble 2.5.6 release of MoveIt.

It does not seem like any new releases of `moveit_msgs` and `moveit_resources` are needed for this one.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/moveit/moveit2/blob/main/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://moveit.picknik.ai/humble/doc/examples/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/moveit/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by sea-bass** [comment]
@rr-tom-noble (or @TSNoble) -- FYI

**Review by mikeferguson — APPROVED** [review]
I presume you then manually also have to push the 2.5.6 tag to github after merging this?

**Comment by sea-bass** [comment]
> I presume you then manually also have to push the 2.5.6 tag to github after merging this?

Exactly! I prefer doing this through a PR to get a second pair of eyes.

**Pull request merged**
